w3sper.js/src/protocol-driver/mod.js

```diff
@@ -349,9 +349,7 @@ export const accountsIntoRaw = async (users) =>
     { memcpy },
   ) {
     let buffer = new Uint8Array(
-      DataBuffer.from(
-        DataBuffer.flatten(users.map((user) => user.account.valueOf())),
-      ),
+      DataBuffer.from(DataBuffer.flatten(users.map((user) => user.valueOf()))),
```

Comment:
Maybe it's a good moment to rename the `users` variable as they are accounts right now?

Reply:
Remember the golden rule of w3sper: ignore everything is under `protocol-driver` for the time being.